asyncapi-2025-goal.md

```diff
@@ -0,0 +1,118 @@
+Carrying on from Thulie, it's time to have an open discussion on the goals for building and maintaining the AsyncAPI community. The Community Manager needs to ensure the goals are addressed and that each program/initiative has a dedicated person responsible for it.
+
+Things I think we should focus on in 2025:
+
+- Increase Maintainers
```

Comment:
but still thinking on retention of existing maintainers - the grant is one of possible solutions